CMakeLists.txt

```diff
@@ -545,7 +547,7 @@ if(USE_CRYPTOMINISAT)
   # CryptoMiniSat requires pthreads support
   set(THREADS_PREFER_PTHREAD_FLAG ON)
   find_package(Threads REQUIRED)
-  find_package(CryptoMiniSat 5.8 REQUIRED)
+  find_package(CryptoMiniSat 5.11.2 REQUIRED)
```

Comment:
Perhaps the `INSTALL.rst` file should be updated to specify the minimum required version;
https://github.com/cvc5/cvc5/blob/main/INSTALL.rst#cryptominisat-optional-sat-solver

Reply:
We didn't before, but yes, maybe we should.